src/main/daml/Daml/Finance/Instrument/Option/Lifecycle/DivOptionRule.daml

```diff
@@ -0,0 +1,99 @@
+-- Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+module Daml.Finance.Instrument.Option.Lifecycle.DivOptionRule
```

Comment:
If this module is called `DivOptionRule` then we should call the other rule an `Option` rule rather than a generic `Rule`

Reply:
Also, it looks like an exact copy of the other rule, shall we use only one?